app/upgrades/vNEXT.go

```diff
@@ -41,9 +46,57 @@ const (
 	Upgrade_NEXT_PlanName = "vNEXT"
 )
 
+// generated via: ./tools/scripts/migration/collect_non_existent_morse_output_accounts.sh --defaults
+var mainNetZeroBalanceMorseClaimableAccountsJSONBZ = []byte(`[
```

Comment:
Do you mind adding some comments of how we can verify/test this during the protocol upgrade process?

E.g.

```
Can be verified onchain after a protocol upgrade by running:

pocketd query migration show-morse-claimable-account 0C3B325133D65B6136CD59511CC63F17EF992BE6 --network=main --grpc-insecure=false -o json
```